quinn-udp/benches/throughput.rs

```diff
@@ -1,76 +1,124 @@
 use criterion::{criterion_group, criterion_main, Criterion};
-use quinn_udp::{RecvMeta, Transmit, UdpSocketState};
-use std::cmp::min;
-use std::net::{Ipv4Addr, Ipv6Addr};
-use std::{io::IoSliceMut, net::UdpSocket, slice};
+use quinn_udp::{RecvMeta, Transmit, UdpSocketState, BATCH_SIZE};
```

Comment:
Nit: would be good to do `StdExternalCrate` import blocks here.

Reply:
:+1: 5609a1d3 among others.